notes/src/App.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from "react";
+import { Route } from "react-router-dom";
+import axios from "axios";
+
+import "./App.css";
+import Navigation from "./components/Navigation";
+import NotesList from "./components/NotesList";
+import AddNoteForm from "./components/AddNoteForm";
+import SingleNote from "./components/SingleNote";
+import EditNoteForm from "./components/EditNoteForm";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      notes: [],
+      title: "",
+      textBody: "",
+      utitle: "",
+      utextBody: "",
+      isDeleting: false,
+      search: ""
+    };
+  }
+  componentDidMount() {
+    axios
+      .get("https://fe-notes.herokuapp.com/note/get/all")
+      .then(res => this.setState({ notes: res.data }))
+      .catch(err => console.log(err));
+  }
+  componentDidUpdate() {
+    axios
+      .get("https://fe-notes.herokuapp.com/note/get/all")
+      .then(res => this.setState({ notes: res.data }))
```

Comment:
setting state here is causing an infinite loop making infinite amount of ajax request .....